iam-login-service/src/main/java/it/infn/mw/iam/config/IamProperties.java

```diff
@@ -230,6 +231,20 @@ public void setLocation(String location) {
     }
   }
 
+  public static class AarcProfileProperties {
+
+    private String urnNamespace = "geant:iam";
```

Comment:
I notice this now. Why geant here?
I would have infn as a default value.

Reply:
As suggested probably using 'example' is better.

Reply:
waiting for an 'infn' entry, example is listed here:
https://www.iana.org/assignments/urn-namespaces/urn-namespaces.xhtml
then 'example' is a good temporary solution. Fixed.